skycatalogs/utils/sed_tools.py

```diff
@@ -220,13 +220,20 @@ def __init__(self, sed_path, fmt='db'):
         assumptions: that the table name is "SED" and that the columns are
         "wavelength" (units angstroms) and "flux" (units flambda)
         '''
+        if not sed_path:
+            # print a warning message?
+            return None
```

Comment:
Is returning `None` really the right thing here?  From the line above in `skyCatalogs.py`, if this object is returned as `None`, the `sso` object type simply isn't registered, but it seems like it should raise an exception if the needed SED info isn't available instead of silently omitting SSOs.

Reply:
Right now the typical config file includes all object types supported by the code, whether or not they're actually available at the installation.  This will change with the stuff I'm currently working on, but for now the fact that the SED isn't there is often the first sign the code sees that there are no SSOs.  I could add a warning message but would rather not raise an exception.  That would force people to meddle with the configs by hand (to remove the sso section) more than I'd like.